src/common/SPELLS/demonhunter.ts

```diff
@@ -153,11 +153,16 @@ const spells = {
     name: 'Immolation Aura',
     icon: 'ability_demonhunter_immolation',
   },
-  IMMOLATION_AURA_FIRST_STRIKE: {
+  IMMOLATION_AURA_FIRST_STRIKE_DPS: {
     id: 258921,
     name: 'Immolation Aura First Strike',
     icon: 'ability_demonhunter_immolation',
   },
+  IMMOLATION_AURA_BUFF_DPS: {
```

Comment:
Followed the pattern on line 156, and also noticed old comments that used this name.

Reply:
Here the dps once again refered to the fact the spell Id is deferent for havoc vs veng. If this continues to be the case then yeah change to to havoc instead of dps.